pkg/operator/observe_config.go

```diff
@@ -44,6 +47,7 @@ func NewConfigObserver(
 	etcdNamespacedKubeInformers kubeinformers.SharedInformerFactory,
 	operatorConfigClient operatorconfigclientv1alpha1.OpenshiftapiserverV1alpha1Interface,
 	kubeClient kubernetes.Interface,
+	clientConfig *rest.Config,
```

Comment:
did you mean to assign this to c.clientConfig below?

Reply:
Thanks, missed that